apps/files_sharing/lib/Updater.php

```diff
@@ -105,6 +106,7 @@ private static function renameChildren($oldPath, $newPath) {
 		$mountManager = \OC\Files\Filesystem::getMountManager();
 		$mountedShares = $mountManager->findIn('/' . \OC_User::getUser() . '/files/' . $oldPath);
 		foreach ($mountedShares as $mount) {
+			/** @var MountPoint $mount */
```

Comment:
```suggestion
		/** @var MountPoint $mount */
		foreach ($mountedShares as $mount) {
```
Would it be better to type it right when it's declared?

Reply:
afaik it doesn't make a difference